### PR TITLE
Storyteller foreground video/image size 

### LINKED ISSUE #274: Story teller -  Foreground video
Story teller component foreground video size needs to be matched with live site.

Live: https://www.esri.com/en-us/artificial-intelligence/overview
EDS: https://main--esri-eds--esri.aem.live/en-us/artificial-intelligence/overview

Live: 
Screenshot 2024-10-30 at 3 01 36 PM

EDS: 
Screenshot 2024-10-30 at 3 01 46 PM

### PR DESCRIPTION
**Pull request by jsavella**
Normalize foreground image/video size and foreground copy gray background. 

Fix #274

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri-eds--esri.aem.live/
- After: https://storytellerBtnAlign--esri-eds--esri.aem.live/

### CODE REVIEW

File: eds/blocks/storyteller/storyteller.css [hunk 1]
```diff
@@ -262,6 +293,14 @@
     display: block;
   }
 
+  .storyteller .foreground-container picture, .storyteller p:nth-of-type(1) picture {
+    display: block;
+  }
+
+  .storyteller .foreground-container a, .storyteller picture.hide-poster {
+    display: none;
+  }
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:302]: isn't this the same or similar to line 61?

Comment (jsavella) [reply eds/blocks/storyteller/storyteller.css]: Correct. Removed

File: eds/blocks/storyteller/storyteller.css [hunk 2]
```diff
@@ -51,7 +50,15 @@
   padding-inline-start: 0;
 }
 
-.storyteller .foreground-container a, .storyteller .foreground-container picture, .storyteller p:nth-of-type(1) picture, .storyteller picture.hide-poster {
+.storyteller:not(.content-right) div div:nth-of-type(2) .foreground-container .content-wrapper {
```
Comment (twhite313) [inline eds/blocks/storyteller/storyteller.css:53]: Should have noticed this earlier -- can we fix the sins of the past and not use directionally oriented class names? Semantically they fail quickly. Should be `content` or `primary-content` or whatever it is.

Comment (jsavella) [reply eds/blocks/storyteller/storyteller.css]: @twhite313 Removed directional oriented class. Replaced with 'primary-content"

File: eds/blocks/storyteller/storyteller.js [hunk 3]
```diff
@@ -160,7 +168,24 @@ export default async function decorate(block) {
...
+    const h2Tags = block.querySelectorAll('h2');
+    if (h2Tags.length > 1) {
+      setforegroundContainersNoVideo(block);
+      const foregroundWrapper = block.querySelector('.foreground-container');
+      foregroundContentContainer.classList.add('foreground-content');
+      foregroundContent.appendChild(h2Tags[1]);
+      if (pTags.length > 5) {
+        foregroundContent.appendChild(pTags[pTags.length - 1]);
+      } else {
+        foregroundContent.appendChild(pTags[4]);
```
Comment (alexcarol) [inline eds/blocks/storyteller/storyteller.js:181]: Won't this only work when pTags.length === 5? (And in that case, isn't 4 === pTags.length - 1 anyway?)

File: eds/blocks/storyteller/storyteller.js [hunk 4]
```diff
@@ -160,7 +168,24 @@ export default async function decorate(block) {
...
+      }
+      foregroundContentContainer.appendChild(foregroundContent);
+      foregroundWrapper.appendChild(foregroundContentContainer);
+    }
+  }
+
+  if ((pictureTagLeft === null) && (vidUrls.length > 0)) {
```
Comment (alexcarol) [inline eds/blocks/storyteller/storyteller.js:188]: You might want to move to pictureTagLeft === null to its own "if" and then just have an if/else with vidUrls.length check (it should be a little more readable).

**Review by ahenriksen-fedx — APPROVED** [review]

File: eds/blocks/storyteller/storyteller.js (hunk 3 above)
Comment (jsavella) [reply eds/blocks/storyteller/storyteller.js:181]: Simplified, no need for if.

**Review by twhite313 — APPROVED** [review]

**Pull request merged**